dashboard/src/components/Config/AppRepoList/AppRepoList.test.tsx

```diff
@@ -122,23 +154,27 @@ xdescribe("global and namespaced repositories", () => {
     ).toExist();
   });
 
-  it("shows the global repositories with the buttons disabled if the current namespace is other", () => {
+  it("shows the global repositories with the buttons disabled if the current user is not allowed to modify them", () => {
```

Comment:
rather than basing the decision if the buttons should be disabled based on the current namespace, check if the user has permissions to perform the action.